src/warnet/tank.py

```diff
@@ -123,6 +135,18 @@ def status(self) -> RunningStatus:
     def exec(self, cmd: str):
         return self.warnet.container_interface.exec_run(self.index, ServiceType.BITCOIN, cmd=cmd)
 
+    def get_bitcoin_conf(self, nodes: list[str]) -> str:
```

Comment:
In 7f4e28460dd0ff75de3d74eb5816df6f57b3f276

If you re-touch could rename this to `gen_bitcoin_options` or something now?